src/Controller/FestivEllesController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class FestivEllesController extends AbstractController
+{
+    /**
+     * @Route("/festiv'elles", name="festiv_elles")
+     */
+    public function index(): Response
+    {
+        return $this->render('festiv_elles/index.html.twig', [
+            'controller_name' => 'FestivEllesController',
```

Comment:
Is it really useful to have the controller's name in the view ?

Reply:
We will need it later